src/Recommendations/RuleModelRepository.php

```diff
@@ -33,6 +33,43 @@ public function map($item)
         );
     }
 
+    public function getLatestArticle()
+    {
+        $prepared = $this->db->prepare('
+          SELECT * FROM Rules 
+          WHERE Rules.type!="subject" 
+          AND Rules.type!="collection" 
+          AND Rules.type!="podcast-episode" 
+          ORDER BY Rules.published DESC
+          LIMIT 40
```

Comment:
@elrossco22 @giorgiosironi Maybe this query would be better suited as a "truth-y" check for the rule type, so `WHERE Rules.type==="research-article" OR Rules.type==="correction" ...`